.github/workflows/test.yml

```
@@ -31,12 +31,15 @@ jobs:
           auto-activate-base: false
       - name: Install dependencies
         shell: bash -l {0}
-        run: |
-          conda install python=3.12 pytest coverage numpy numba conda-build clangxx=14 numba/label/dev::llvmlite setuptools_scm cython pytest-cov pyyaml
+          conda install python=3.12 pytest coverage numpy numba flake8 conda-build clangxx=14 numba/label/dev::llvmlite setuptools_scm cython pytest-cov pyyaml
```

```suggestion
       run: |
          conda install python=3.12 pytest coverage numpy numba flake8 conda-build clangxx=14 numba/label/dev::llvmlite setuptools_scm cython pytest-cov pyyaml
```
is this now missing a `run` key?